README.md

```
@@ -1,12 +1,12 @@
 # ZipCaptions
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)